server/athenian/api/controllers/miners/github/pull_request.py

```diff
@@ -95,26 +97,41 @@ def _deserialize_from_cache(data: bytes) -> List[pd.DataFrame]:
             dfs.append(df)
         return dfs
 
+    def _delete_blacklisted_prs(result: List[pd.DataFrame],
+                                pr_blacklist: Optional[Collection[str]] = None,
+                                **_) -> List[pd.DataFrame]:
```

Comment:
@se7entyse7en I am planning to filter by repository and developer here (the function is to be renamed ofc).